client/swagger.yaml

```diff
@@ -353,19 +353,14 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-  /vsocks/{id}:
+  /vsocks:
```

Comment:
Just noticed there is a discrepancy, the swagger.yaml on the v0.19 release branch of Firecracker calls this `/vsock` but here it is `/vsocks`: https://github.com/firecracker-microvm/firecracker/blob/effaab05e4b02b25c578273d966e04c98d2cf2e1/api_server/swagger/firecracker.yaml#L356-L381

Reply:
Yep, also noticed that, fix is here already.